Management.Database/Stores/ManaSqlVoteStore.cs

```diff
@@ -51,16 +51,22 @@ public ManaSqlVoteStore(string connectionString) : base(connectionString)
             return vote.First();
         }
 
-        public async Task<IEnumerable<ManaVote>> GetUserVotes(int userId, string state = "all")
+        public async Task<IEnumerable<ManaVote>> GetAllUserVotes(int userId)
+        {
+            var vote = await GetVoteByQuery(ManagementQueries.VotesGetByUser, new { UserId = userId });
+            return vote;
+        }
+
+        public async Task<IEnumerable<ManaVote>> GetUserVotes(int userId, bool published)
         {
             dynamic vote;
-            if (state == "all")
+            if (published)
```

Comment:
Yeah, I left it in when the state `all` could be given, meaning a different query would have to be run. I took that out into its own route as part of this though, so only one query will be needed here. Fixed in 48a2eff606bdfc1034fccac18ed178f01fec39da.